src/pages/Loans/LoansOverview/components/BorrowLimit/BorrowLimit.tsx

```diff
@@ -32,9 +32,16 @@ const BorrowLimit = ({
   const { t } = useTranslation();
 
   const { data: currentBorrowLimit, getBorrowLimitUSD } = useAccountBorrowLimit();
+  // ray test touch <<
   const { data: currentLTV, getLTV } = useGetLTV();
+  console.log('ray : ***** currentLTV => ', currentLTV);
+  console.log('ray : ***** currentBorrowLimit?.toString() => ', currentBorrowLimit?.toString());
+  // ray test touch >>
 
+  // ray test touch <<
+  // TODO: update with `calculateBorrowLimitBtcChange`
   const newBorrowLimit = getBorrowLimitUSD({ type: loanAction, amount: actionAmount, asset });
```

Comment:
This should be refactored using `calculateBorrowLimitBtcChange` from `LoanCollateralInfo`. Check `useLoanInfo` where you can get `LoanCollateralInfo`.